lib/travis/client/session.rb

```diff
@@ -17,7 +17,7 @@ module Travis
   module Client
     class Session
       PRIMITIVE   = [nil, false, true]
-      SSL_OPTIONS = { :ca_file => Tools::Assets['cacert.pem'] }
```

Comment:
Would it be possible to remove this tls option only when running in enterprise mode?